app/controllers/api/v3/statements_controller.rb

```diff
@@ -1,8 +1,10 @@
 module API
   module V3
     class StatementsController < BaseController
+      include Pagination
```

Comment:
maybe we can incorporate handing wrong page parameters as well, I think now in ECF if you pass in a string it throws an error?

Reply:
@cwrw added error rescue for wrong page parameter.